modules/api-svc/package.json

```diff
@@ -67,12 +67,12 @@
     "@mojaloop/api-snippets": "17.7.4",
     "@mojaloop/central-services-error-handling": "^13.0.2",
     "@mojaloop/central-services-logger": "^11.5.1",
-    "@mojaloop/central-services-metrics": "^12.0.8",
-    "@mojaloop/central-services-shared": "18.11.0",
+    "@mojaloop/central-services-metrics": "^12.1.0",
+    "@mojaloop/central-services-shared": "18.11.1",
     "@mojaloop/event-sdk": "^14.1.1",
     "@mojaloop/ml-schema-transformer-lib": "2.3.4",
     "@mojaloop/sdk-scheme-adapter-private-shared-lib": "workspace:^",
-    "@mojaloop/sdk-standard-components": "^19.4.1",
+    "@mojaloop/sdk-standard-components": "19.5.0-snapshot.12",
```

Comment:
putting here a reminder to merge @mojaloop/sdk-standard-components first

Reply:
done